eng/scripts/build.ps1

```diff
@@ -29,7 +30,7 @@ function Process-Sdk ()
     if ($cleanGenerated)
     {
         Write-Host "##[command]Cleaning auto-generated files in" $currentDirectory
-        Remove-Item "zz_generated_*"
+        Get-ChildItem "*.go" | Select-String -Pattern "Code generated by Microsoft (R) AutoRest Code Generator." | Select-Object -ExcludeProperty path | Remove-Item -Force -Confirm
```

Comment:
Yes, this will delete auto-generated test file. At first, I think it's OK to delete them all. But now I change my mind. It could be clearer to only delete what this script generates, and we have test generation script to deal with the test.